src/System.Transactions/src/System/Transactions/Transaction.cs

```diff
@@ -306,6 +306,18 @@ internal Transaction(IsolationLevel isoLevel, ISimpleTransactionSuperior superio
             _cloneId = 1;
         }
 
+        // This serializtion constructor needs to be removed when we implement distributed transaction capabilities
+        // Issue 13137
+        protected Transaction(SerializationInfo serializationInfo, StreamingContext context)
```

Comment:
This should probably be internal, not protected, as you don't want to expose this ctor out of the assembly, especially if you plan to remove it.